src/components/data-pane/field-list.tsx

```diff
@@ -65,36 +59,39 @@ class FieldListBase extends React.PureComponent<FieldListProps, {}> {
     });
   }
 
-  private renderComponent(fieldDef: ShelfFieldDef, hideTypeChanger: boolean, primitiveType: PrimitiveType) {
-    if (hideTypeChanger) {
-      return this.renderField(fieldDef);
-    } else {
-      const popupComponent = this.renderTypeChanger(fieldDef, primitiveType);
-      return this.renderField(fieldDef, popupComponent);
-    }
-  }
+  private renderListItem(fieldDef: ShelfFieldDef, index: number) {
+    const {schema, filters, handleAction} = this.props;
+    const key = isWildcard(fieldDef.field) ? stringify(fieldDef) : fieldDef.field;
+
+    let field;
+    let popupComponent;
+    let isCustomWildcardField;
```

Comment:
Why do you need `let` for `isCustomWildcardField`?  You can just use `const` here.